extensions/GlobalMessages/GlobalMessagesT.i18n.php

```diff
@@ -31611,100 +31774,99 @@
-	'thumbnail_gd-library' => 'Configuração da biblioteca GD incompleta: função $1 em falta',
-	'thumbnail_image-missing' => 'Ficheiro em falta: $1',
-	'tooltip-pt-userpage' => 'A sua página de utilizador',
-	'tooltip-pt-anonuserpage' => 'A página de utilizador para o endereço IP que está a usar',
-	'tooltip-pt-mytalk' => 'A sua página de discussão',
-	'tooltip-pt-anontalk' => 'Discussão sobre edições feitas a partir deste endereço IP',
-	'tooltip-pt-preferences' => 'Configuração dos comportamentos que prefere da wiki',
-	'tooltip-pt-watchlist' => 'Lista de mudanças nas páginas que está a vigiar',
-	'tooltip-pt-mycontris' => 'Lista das suas contribuições',
-	'tooltip-pt-login' => 'É encorajado a autenticar-se, apesar de não ser obrigatório.',
-	'tooltip-pt-anonlogin' => 'É encorajado a autenticar-se, apesar de não ser obrigatório.',
-	'tooltip-pt-logout' => 'Terminar esta sessão na wiki',
+	'thumbnail_gd-library' => 'Configuração da biblioteca GD incompleta: função $1 não encontrada',
+	'thumbnail_image-missing' => 'Arquivo aparentemente inexistente: $1',
+	'tooltip-pt-userpage' => 'Sua página de usuário',
+	'tooltip-pt-anonuserpage' => 'A página de usuário para o ip com o qual você está editando',
+	'tooltip-pt-mytalk' => 'Sua página de discussão',
+	'tooltip-pt-anontalk' => 'Discussão sobre edições deste endereço de ip',
+	'tooltip-pt-preferences' => 'Suas configurações',
+	'tooltip-pt-watchlist' => 'Lista de alterações nas páginas que você está monitorando',
+	'tooltip-pt-mycontris' => 'Listagem de suas contribuições',
+	'tooltip-pt-login' => 'Você é encorajado a autenticar-se, apesar disso não ser obrigatório.',
+	'tooltip-pt-anonlogin' => 'Você é encorajado a autenticar-se, apesar disso não ser obrigatório.',
+	'tooltip-pt-logout' => 'Sair',
 	'tooltip-ca-talk' => 'Discussão sobre o conteúdo da página',
-	'tooltip-ca-edit' => 'Pode editar esta página.
-Utilize o botão "Antever resultado" antes de gravar, por favor.',
-	'tooltip-ca-addsection' => 'Iniciar uma nova secção',
-	'tooltip-ca-viewsource' => 'Esta página está protegida; só pode ver o conteúdo.',
+	'tooltip-ca-edit' => 'Você pode editar esta página. Use o botão "Mostrar previsão" antes de salvar.',
+	'tooltip-ca-addsection' => 'Iniciar uma nova seção',
+	'tooltip-ca-viewsource' => 'Esta página está protegida.
+Você pode, no entanto, visualiar seu código-fonte.',
 	'tooltip-ca-history' => 'Edições anteriores desta página.',
 	'tooltip-ca-protect' => 'Proteger esta página',
-	'tooltip-ca-unprotect' => 'Alterar a protecção desta página',
+	'tooltip-ca-unprotect' => 'Alterou a proteção desta página',
 	'tooltip-ca-delete' => 'Apagar esta página',
 	'tooltip-ca-undelete' => 'Restaurar edições feitas a esta página antes da eliminação',
 	'tooltip-ca-move' => 'Mover esta página',
-	'tooltip-ca-watch' => 'Adicionar esta página à lista de páginas vigiadas',
+	'tooltip-ca-watch' => 'Adicionar esta página à sua lista de páginas vigiadas',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Adicionar esta página à sua lista de páginas vigiadas" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)